tutorials/optical_tactile_plugin.md

```diff
@@ -171,6 +182,86 @@ is set to `true` and this value is too high, it may slow down the simulation.
 - `<clip>`: These values should be set in a way that the contact surface stays
 in between its far and near values. Otherwise, nothing will be visualized.
 
+## Topics & services
+In order to access the data computed by the plugin and control it during
+simulation, the following topics and services have been advertised:
+
+  `/<namespace>/enable` : Service used to enable and disable the plugin. If disabled,
+  the plugin won't publish messages in the `/normal_forces` nor the `/contacts` topics.
+
+  `/<namespace>/normal_forces` : Topic where a message is published each
+  time the normal forces are computed.
+
+  `/<namespace>/contacts` : Topic where a message is published each time the contacts
+  are computed.
+
+## Performance
+In order to get an idea about the resources consumed by the plugin, some
+experiments have been made by setting the parameters to specific values.
```

Comment:
```suggestion
In order to give an idea about the resources consumed by the plugin, we carried 
out some simple experiments by setting the parameters to specific values.
```
So, the caveat with testing performance is that, part of the performance is dependent on Ignition's efficiency, and part of is dependent on hardware specifications. While Ignition's own efficiency will determine a max performance ceiling, high-performing hardware can have a higher ceiling than lower-performing hardware. So I would suggest specifying the hardware this is tested on. Specifying the CPU should be enough, e.g. laptop with Intel blah.

Reply:
7b40828